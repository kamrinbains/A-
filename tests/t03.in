5
0 0 0
1 1 0
2 5 0
3 6 0
4 7 0
2
0 1 1.0
3 4 1.0
0
4
1
