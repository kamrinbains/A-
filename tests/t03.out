No path found from 0 to 4
